### ISSUE TITLE
Nothing showing in Debian 8 using Terminator and patched fonts

### ISSUE THREAD

**Issue by f3bruary** [illinoisjackson/even-better-ls#21]
I installed it successfully but there is nothing showing in ls.
I also installed it on a server (also Debian) and it works fine there. Both using zsh.

No idea where to start debugging. Perhaps you can point me in the right direction ?

**Comment by heyitsols** [comment]
Does running `ls_colors_generator` output the font icons?

**Comment by f3bruary** [comment]
nope, seems like gibberish with color containing file extensions here and there.

**Comment by Noammac** [comment]
Are you sure you changed Terminator's font?

**Comment by f3bruary** [comment]
Yes. It is working in an ssh shell after all (also using terminator). 

Does $TERM influence this ?

**Comment by Noammac** [comment]
Can you show your Terminator configuration?

I don't think $TERM influences this.

**Comment by f3bruary** [comment]
Sure, although there's not much in there that can cause this. I use the same terminal emulator the the ssh session after all.

[global_config]
  use_custom_url_handler = True
  custom_url_handler = /usr/bin/chromium --incognito
  enabled_plugins = LaunchpadCodeURLHandler, APTURLHandler, LaunchpadBugURLHandler
[keybindings]
[profiles]
  [[default]]
    foreground_color = "#cacaca"
    palette = "#000000:#aa0000:#00aa00:#aa5500:#0000aa:#aa00aa:#00aaaa:#aaaaaa:#555555:#ff5555:#55ff55:#ffff55:#5555ff:#ff55ff:#55ffff:#ffffff"
    font = Inconsolata-g for Powerline Medium 10
    background_image = /home/user/b6yfmnE.png
    background_darkness = 0.77
    use_system_font = False
    cursor_color = "#0087af"
    scroll_background = False
    scrollbar_position = hidden
    show_titlebar = False
    copy_on_selection = True
    background_color = "#1a1f28"
    allow_bold = False
[layouts]
  [[default]]
    [[[child1]]]
      parent = window0
      type = Terminal
    [[[window0]]]
      parent = ""
      type = Window
[plugins]

**Comment by Noammac** [comment]
Does the font section work with other fonts? Try it with:
a) A non-powerline font
b) A powerline font

**Comment by f3bruary** [comment]
I've tried this. All fonts give me the same gibberish when running ls_colors_generator

EDIT: Wanted to add that now both in ssh as local shells show no icons. Dunno what I am doing wrong. I've tried re-installing but it's still the same.

**Comment by f3bruary** [comment]
Any ideas ? I'd really like to get this working. I've ran install.sh and reinstalled nerd fonts. Still same issue.

**Comment by f3bruary** [comment]
I've discovered something interesting. In zsh (oh-my-zsh) when I enter interactive autocompletion (e.g. ls <tab>) I see icons !

Some seem a bit off though, like there's another icon underneath the one showing.
